src/lib/y2security/lsm/config.rb

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) [2021] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "yast2/execute"
+require "y2security/lsm/base"
+require "y2security/lsm/app_armor"
+require "y2security/lsm/none"
+require "y2security/lsm/selinux"
+
+Yast.import "Stage"
+Yast.import "ProductFeatures"
+
+module Y2Security
+  module LSM
+    # This class allows to check and select one of the supported Linux Security Modules (LSM)
+    class Config
+      include Yast::Logger
+      include Singleton
+
+      RUNNING_PATH = "/sys/kernel/security/lsm".freeze
+      SUPPORTED = [None, Selinux, AppArmor].freeze
+
+      # @return [AppArmor,Selinux, nil] selected module
```

Comment:
```suggestion
      # @return [AppArmor, Selinux, nil] selected module
```

Could it be _None_ too?